src/components/TrackTile/hooks/useAxiosTrackTileService.ts

```diff
@@ -67,6 +71,19 @@ export const useAxiosTrackTileService = (): TrackTileService => {
     }
   }, [userId, cache, previousUserId]);
 
+  // Clear cached tracker values during HomeScreen refresh
+  useEffect(() => {
+    const refreshHandler = (refreshParams: RefreshParams) => {
+      if (refreshParams.context === 'HomeScreen') {
+        cache.trackerValues = {};
```

Comment:
Is the thinking that trackers and ontologies won't change enough to warrant clearing the cache?

Reply:
Yep, that's exactly what I landed on.  I could see this becoming config-based if we need a deeper refresh in some cases.